data/json/snippets/effect_on_conditions.json

```
@@ -0,0 +1,40 @@
+[
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_1",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  The stars are quite dense and are precisely aligned in several regular polyhedrons next to each other.",
+      "You see what seems to be a new constellation in the sky.  The stars are in a precise spiral, with the central star constantly shifting color.",
+      "You see several shooting stars.  They seem larger than normal, and are all a deep purple.",
+      "You see a shooting star.  It is bright red and does not fall in a straight line.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon, but square and green.",
+      "For a brief moment, the stars and moon all disappear.  When they come back, you're not sure if they're in the same places as they were."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_2",
+    "text": [
+      "You see what seems to be a new constellation in the sky.  It changes shape as you look from a triangle to a pentagon.",
+      "You see what seems to be a new constellation in the sky, in the shape of a hexagon.  It's easy to pick out - there are glowing lines between the stars.",
+      "You see several shooting stars.  After they go below the horizon a bright glow briefly appears.",
+      "You see a shooting star.  It is bright red and stops falling after a second to hover there ominously.",
+      "For a brief moment you see a shape in the sky roughly the size of the moon.  It had almost looked like a blinking eye.",
+      "You see a moving object in the sky with a blinking red light coming from it."
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "NIGHT_MESSAGES_3",
+    "text": [
+      "You see a once-familiar constellation and watch as several of its stars move.",
+      "You see what seems to be a new constellation in the sky.  It reminds you of a serpent - it moves like one.",
+      "You see a shooting star.  It seems to be heading straight for you before vanishing suddenly.",
+      "For a brief moment, you see a pentagonal shape in the sky, roughly twice the size of the moon.  It splits into several triangles and then reforms before vanishing.",
+      "As you look at the stars, they wink out for an instant, reappearing just slightly out of place, as if the night sky had been rotated ever so slightly. This occurs twice more, in rapid succession.",
+      "A strange cloud seems to float in the sky, at the very edge of your peripheral vision. Just as you shift your gaze to it, it evaporates, boiling away into a stream of shooting stars.",
+      "For a brief moment, you feel like you've been jerked to the side. You haven't moved, though -- the stars have.",
```

```suggestion
      "For a brief moment, you feel like you've been jerked to the side. You haven't moved, though - the stars have.",
```
Or the above ones should use a double dash (IMO using a single one should be in the MANUAL_OF_STYLE.md but that's probably just me...).

---

One or the other is good -- we just need to pick a standard and go over the entire codebase for consistency.